### DISCUSSION TITLE
cigale 100Myr SFRs seem to have some upper limit constraints

### DISCUSSION THREAD

**Discussion by aswinpvijayan** [mboquien/cigale#53] [Q&A] [answered]
I am new cigale sed fitting, and have been using it to fit some dusty star forming galaxy data while including some JWST points as well. I have also been using this other sed fitting code, stardust (https://github.com/VasilyKokorev/stardust), for doing comparisons. One of the comparisons I have made is to plot the star forming sequence, comparing the 100Myr SFRs. From looking at the plot it seems like the SFRs have some sort of cap in my cigale fitting for a given stellar mass, as the points seem to follow a line (attached plot). Do you have any suggestions on why this could be happening (pcigale.ini file attached)? Is there some normalisation I am missing?
[pcigale.ini.txt](https://github.com/mboquien/cigale/files/11380837/pcigale.ini.txt)
![MS_cigale_stardust_0_vnorm_sfh2exp](https://user-images.githubusercontent.com/32305846/235869281-48fc0b55-b17b-4b7b-b6f1-86c6d6c965cc.png)

**Comment by mboquien** [comment, marked as answer]
The choice of the parametrization will have some effect on the SFR vs Mstar relation. See for instance https://ui.adsabs.harvard.edu/abs/2017A%26A...608A..41C/abstract . You may want to allow stronger burst if you want to reach higher SFR. This being said, the relation you obtain with cigale already looks quite reasonable to me overall.

**Reply by aswinpvijayan** [reply]
Thanks, I will try and include stronger bursts and see what happens. 
I was concerned by that almost linear relationship between the stellar mass and the SFR, and thought there might be something going wrong.

**Reply by mboquien** [reply]
A strong relation between the two quantities is expected for star-forming galaxies, so that looks mostly fine.
